v4/source/groups.yaml

```diff
@@ -58,6 +58,15 @@
           required: true
           schema:
             type: string
+        - name: since
+          in: query
+          description: >
+            Only return groups that have been modified since the given Unix
+            timestamp (in milliseconds).
+
```

Comment:
Can we update the documentation to indicate that this will cause the server to return even the deleted groups.